earth_enterprise/src/fusion/config/geconfigureassetroot.cpp

```diff
@@ -388,6 +404,42 @@ void RepairExistingAssetRoot(const AssetRootStatus &status, bool noprompt) {
 }
 
 // ****************************************************************************
+// ***  ListVolumes
+// ****************************************************************************
+void listCurrentVolumes(VolumeDefList *const voldefs, std::vector<std::string> volnames) {
+    printf("\nCurrent Volume Definitions:\n");
```

Comment:
@tst-ppenev Unchanged code (just refactored to be shared method, avoiding duplication). Out of scope for this ticket.